src/main/java/com/kat/bachaat/model/User.java

```diff
@@ -0,0 +1,110 @@
+package com.kat.bachaat.model;
+
+import java.util.Random;
+
+public class User {
+    private Long userId;
+    private String firstName;
+    private String middleName;
+    private String lastName;
+    private String email;
+    private String address;
+    private int activationCode;
+    private Long mobileNumber;
+    private boolean active = false;
+
+    public User() {
+        this.activationCode = new Random().nextInt(900000) + 100000;
+    }
```

Comment:
Generating activation code is not encourage here

Reply:
Okay, I will remove from here and place it somewhere.